### PR TITLE
netlify adapter docs: add warning on redirects file placement

### PR DESCRIPTION
**Pull request by sw-yx**
### Before submitting the PR, please make sure you do the following

addresses https://github.com/sveltejs/kit/pull/1586 to hopefully save others from the same confusion

### CODE REVIEW

File: packages/adapter-netlify/README.md [hunk 1]
```diff
@@ -42,13 +42,19 @@ You may build your app using functionality provided directly by SvelteKit withou
 
 ### Using Netlify Redirect Rules
 
+> [Netlify docs here](https://docs.netlify.com/routing/redirects/)
+
+Netlify supports declarative redirects via a `_redirects` file placed in your `publish` directory (usually "`build`", as configured in your `netlify.toml` above. NOT in your project root next to `netlify.toml`).
```
Comment (benmccann) [inline packages/adapter-netlify/README.md:47]: the `build` directory is generated by SvelteKit, so I wouldn't think you could put it there. I thought on the last issue it was said to go in `static`?

Comment (roschaefer) [reply packages/adapter-netlify/README.md:47]: Yes! It is `static/` unless you configured `kit.files.assets` to something else in the [project configuration](https://kit.svelte.dev/docs#configuration-files).

Comment (sw-yx) [reply packages/adapter-netlify/README.md:47]: Yeah my bad... 

```suggestion
Netlify supports declarative redirects via a `_redirects` file. Place it in the `static/` folder, NOT in your project root next to `netlify.toml`.
```

**Comment by benmccann** [comment]
This feels quite wrong to me. `_redirects` isn't a static asset. There are three different directories that get copied into the `publish` directory. We'd just be relying on the fact that static assets end up in the `publish` directory, but it's quite an unintuitive place to have to put the file.

I'd rather put it at the root of the project. In neither case would we be putting it directly in the `publish` directory, but rather relying on the adapter to copy it to the right place. It's just a question of where would you most expect it to live. Living next to `netlify.toml` makes more sense than living next to your image files, etc. in my opinion

**Comment by sw-yx** [comment]
i see. so perhaps @roschaefer should reopen #1586 ?

**Comment by kvn-shn** [comment]
At least to me it feels quite intuitive putting it into `static/` (e.g. next to robots.txt) because it's more like an oldschool `.htaccess` file that belongs in the _"public_html" of the server_... After all it _is_, in the context of Svelte, a static asset that Svelte can't and won't do anything with.

So in my opinion this PR would do the trick.

**Comment by benmccann** [comment]
I went ahead and merged https://github.com/sveltejs/kit/pull/1586. It shouldn't stop you from putting it in `static/` if you'd like, but I think supporting it in the root directory next to `netlify.toml` makes a lot of sense

File: packages/adapter-netlify/README.md [hunk 2]
```diff
@@ -42,13 +42,19 @@ You may build your app using functionality provided directly by SvelteKit withou
 
 ### Using Netlify Redirect Rules
 
+> [Netlify docs here](https://docs.netlify.com/routing/redirects/)
+
+Netlify supports declarative redirects via a `_redirects` file. Place it in the `static/` folder, NOT in your project root next to `netlify.toml`.
+
 During compilation a required "catch all" redirect rule is automatically appended to your `_redirects` file. (If it doesn't exist yet, it will be created.) That means:
 
 - `[[redirects]]` in `netlify.toml` will never match as `_redirects` has a [higher priority](https://docs.netlify.com/routing/redirects/#rule-processing-order). So always put your rules in the [`_redirects` file](https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file).
```
Comment (sw-yx) [inline packages/adapter-netlify/README.md:51]: just fyi - i think this is inaccurate - the doc actually says "Rules in the _redirects file are always processed first, followed by rules in the Netlify configuration file."

in other words you CAN put stuff in netlify.toml, AND in _redirects, its just that the latter overrides the former in cases of conflict.

Comment (benmccann) [reply packages/adapter-netlify/README.md:51]: I believe it's true when you consider the catch-all route that gets added

**Pull request closed without merging**